tools/swift_stdlib_tool/swift_stdlib_tool.sh

```diff
@@ -124,4 +124,10 @@ readonly TEMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/swift_stdlib_tool.XXXXXX")
 trap 'rm -rf "$TEMP_DIR"' EXIT
 
 copy_swift_stdlibs "$TEMP_DIR"
+echo "$TEMP_DIR"
+if ! ls "$TEMP_DIR/libswift"* > /dev/null 2>&1; then
+  echo "error: No swift libraries were copied, please report this on https://github.com/bazelbuild/rules_apple/issues" >&2
```

Comment:
can you change this to 

`error: Expected to copy Swift Standard Lib dylibs, but none were copied.`

given this code will run in other contexts, let's avoid having a mechanism for filing issues